git_machete/github.py

```diff
@@ -36,7 +89,7 @@ def fire_github_api_get_request(url: str, token: Optional[str]) -> Any:
     if token:
         headers['Authorization'] = 'Bearer ' + token
 
-    host = 'api.github.com'
+    host = 'api' + GITHUB_DOMAIN
```

Comment:
Whoops shouldn't it be:
```suggestion
    host = 'api.' + GITHUB_DOMAIN
```